microsoft-edge/devtools-guide-chromium/css/reference.md

```diff
@@ -325,7 +325,7 @@ To instead add the rule in an existing stylesheet:
 
 By default, adding a style rule by clicking on **New Style Rule** inserts the new rule beneath the **element.style** rule in the `inspector-stylesheet` stylesheet.
 
-To add a style rule in a specific location of the **Styles** panel instead:
+To add a style rule in a specific location of the **Styles** pane instead:
```

Comment:
No change proposed here; a style comment/idea:
* When need to envision & put focus on selecting/clicking the little tab in the tab-set, say "tab".
* When putting focus on content that's inside the entire tab ("tab" in the encompassing, broad sense), say "pane".

Depends on what you have in mind: the little tab to click, vs. the entire tab/panel that's accessed via the little tab.